charms/kserve-controller/src/templates/auth_manifests.yaml.j2

```diff
@@ -363,6 +384,7 @@ metadata:
     app.kubernetes.io/name: {{ app_name }}
     rbac.authorization.kubeflow.org/aggregate-to-kubeflow-admin: "true"
   name: kubeflow-kserve-admin
+rules: []
```

Comment:
```suggestion
```
I think including this line actually raises an error in Lightkube during apply.  It is functionally equivalent if we leave it omitted.